src/types/index.ts

```diff
@@ -116,3 +116,36 @@ export interface CreatePullRequest {
   // Whether or not maintainers can modify the PR.
   maintainersCanModify: boolean;
 }
+
+export class RawContent {
```

Comment:
These look like simple value classes. Can they just be interfaces instead?

Reply:
I could. The trade-off is the conciseness
i.e.
```
const patch = new Patch(1,2,'hello');
```
vs
```
const patch: Patch = { start: 1, end: 2, content: 'hello' }
```
I'll update this in a future PR to use interfaces to maintain the logical nuance of a class over the conciseness